setup.py

```diff
@@ -1,17 +1,15 @@
 import os
 from setuptools import setup
 
-SITE_PACKAGES = '/usr/lib/python2.7/site-packages'
-DIST_PACKAGES = '/usr/lib/python2.7/dist-packages'
+from distutils import sysconfig
+site_packages_path = sysconfig.get_python_lib()
 
-
-data_files = []
+data_files = [(site_packages_path, ["conf/upseto.pth"])]
```

Comment:
This will be taken care of by get_python_lib() function --- on venv it will eval correctly.